artifacts/definitions/Admin/Client/Uninstall.yaml

```diff
@@ -26,15 +26,15 @@ parameters:
 
 sources:
   - precondition:
-      SELECT OS From info() where OS = 'windows'
+            SELECT OS From info() where OS = 'windows'
 
     query:  |
-      LET packages = SELECT Name, DisplayName FROM Artifact.Windows.Sys.Programs()
+      LET packages = SELECT Name, DisplayName,UninstallString FROM Artifact.Windows.Sys.Programs()
       WHERE DisplayName =~ DisplayNameRegex AND
         log(message="Will uninstall " + DisplayName)
+      
+      LET uninstall(DisplayName) = SELECT * FROM execve(argv=[commandline_split(command=UninstallString) + "/quiet"])
```

Comment:
Does this work on multiple returned results?

I think you would need to make this something like:
      LET uninstall(uninstall_string) = SELECT * FROM execve(argv=[commandline_split(command=uninstall_string) + "/quiet"])

Reply:
No this does not work because then it will create an array where the first member is also an array - we need an array of strings.